### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by tizmagik**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - 6
-script: npm run lint && npm test && npm run build
+script:
+  - npm run lint
+  - npm test
+  - npm run build
```
Comment (TrySound) [inline .travis.yml]: I would handle it through npm scripts like
```
"pretest": "npm run lint",
"test": "..."
"posttest": "npm run build"
```

File: package.json [hunk 2]
```diff
@@ -61,11 +69,20 @@
...
+      "./src"
+    ],
+    "setupFiles": [
+      "raf/polyfill",
+      "./utils/setup"
+    ]
   },
   "lint-staged": {
-    "*.{js,json}": ["prettier --write", "git add"]
+    "*.{js}": [
```
Comment (TrySound) [inline package.json]: remove brackets?

File: package.json [hunk 3]
```diff
@@ -69,24 +61,16 @@
...
+    "roots": ["./src"],
+    "setupFiles": ["raf/polyfill", "./utils/setup"]
   },
   "lint-staged": {
-    "*.{js}": [
-      "prettier --write",
-      "npm run lint -- --fix",
-      "git add"
-    ]
+    "*.{json,js}": ["eslint", "prettier --write", "git add"]
```
Comment (TrySound) [inline package.json]: add .eslintignore in prettier here

File: .travis.yml (hunk 1 above)
Comment (tizmagik) [reply .travis.yml]: Yea, fair!

**Pull request merged**